src/internal/uuid/uuid.go

```diff
@@ -42,4 +42,4 @@ func IsUUIDWithoutDashes(s string) bool {
 // Because we use UUIDv4, the 13th character is a '4'.
 // Moreover, a UUID can only contain "hexadecimal" characters,
 // lowercase here.
-var uuidWithoutDashesRegexp = regexp.MustCompile("[0-9a-f]{12}4[0-9a-f]{19}")
+var uuidWithoutDashesRegexp = regexp.MustCompile("^[0-9a-f]{12}4[0-9a-f]{19}$")
```

Comment:
This regex wasn't anchored, so it would return `true` for something like `repo@<uuid>`, which is obviously wrong.  This _shouldn't_ impact any existing code, but it's possible someone is relying on this broken behavior.